client/components/IndividualChallenge.js

```diff
@@ -84,9 +96,12 @@ const _IndividualChallenge = props => {
 const mapDispatchToProps = dispatch => ({
   putUserchallenge: (userAnswer, userchallengeId, isSubmit) =>
     dispatch(putUserchallenge(userAnswer, userchallengeId, isSubmit)),
```

Comment:
Obviously need to change the name to `updateUserchallenge`  in the store and also elsewhere in this file.
```suggestion
    dispatch(updateUserchallenge(userAnswer, userchallengeId, isSubmit)),
```